src/main/java/uk/gov/ons/census/casesvc/model/entity/Case.java

```diff
@@ -114,4 +114,7 @@
   @Column(columnDefinition = "timestamp with time zone")
   @UpdateTimestamp
   private OffsetDateTime lastUpdated;
+
+  @Column(columnDefinition = "BOOLEAN")
```

Comment:
you don't need the `columnDefinition` for the primative/common types

Reply:
I notice `@Column(nullable = false, columnDefinition = "BOOLEAN DEFAULT false")` was added rather over-zealously in this entity file. It's only `Boolean` type which needs to be told it's not nullable and given a default. The primitive `boolean` is infinitely preferable for values which are non-nullable.